200.blacklists/metaports.yaml

```diff
@@ -4,6 +4,9 @@
 
 - { remove: true,   category: virtual, ruleset: [exherbo,gentoo] }
 - { remove: true,   category: java-virtuals, ruleset: [exherbo,gentoo] }
+- { remove: true,   category: acct-group, ruleset: gentoo }
```

Comment:
You don't need `acct-*` here as these are ignored in the parsers

Reply:
Ok, updated.